### PR TITLE
Added changelog for Evals detailed view

### PR DESCRIPTION
**Pull request by olehvell-h**
Changelog for https://github.com/humanloop/hl-alpha/pull/4105

(V2 after Harry's and Jordan's comments)  

image

### CODE REVIEW

File: fern/changelog/2024-10-02.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Evals Comparison Mode
+
+We've added Comparison Mode to the Evals page. The new view enables Human Evaluators to compare multiple outputs side-by-side, making it easier to evaluate and compare different model outputs.
+
+![Comparison Mode in Evals](../assets/images/changelogs/evals_comparison_mode.png)
+
+Evaluating model outputs is crucial part of shipping reliable AI features. You can read how Humanloop helps you with evaluations in our [documentation](https://humanloop.com/docs/evaluation/guides/run-evaluation).
```
Comment (PeadarOhAodha) [inline fern/changelog/2024-10-02.md:7]: english nit - is a crucial part

File: fern/changelog/2024-10-02.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Evals Comparison Mode
+
+We've added Comparison Mode to the Evals page. The new view enables Human Evaluators to compare multiple outputs side-by-side, making it easier to evaluate and compare different model outputs.
+
+![Comparison Mode in Evals](../assets/images/changelogs/evals_comparison_mode.png)
```
Comment (PeadarOhAodha) [inline fern/changelog/2024-10-02.md:5]: Is this placeholder screenshot?

Has unfortunate scroll bars and i would also hide the details of the side panel to focus most on the new feature

File: fern/changelog/2024-10-02.md (hunk 1 above)
Comment (olehvell-h) [reply fern/changelog/2024-10-02.md:7]: it's a crucial part

**Comment by harry-humanloop** [comment]
image


Changelog*

(textedit on my mac keeps making this correction too)

**Review by harry-humanloop — COMMENTED** [review]
quick comments:

- make the window in the screenshot less wide. currently there's a lot of whitespace in the human evaluators panel which looks bad, and the first column looks too thin (which highlights the lack of resizing here!)
- We try to follow a structure of "what is it, why do i care, how do i use it" (questions from a user's pov that we have to answer). The "what is it" is mostly here (side by side view). do add that we've improved the controls and have made it easier to add judgments for all human evaluator types. the "why do i care" is kinda here, but should be extended - easier for domain experts to focus on annotating and reviewing model generations.


- the text says "compare" too many times. (I'd actually push to mention "compare" as few times as possible, as you can't really provide a "comparison judgment" (e.g. "left is better"). i think we can avoid mentioning "compare" and stick to the factual "allows you to view generations from multiple versions of your prompt". but i suspect others have a stronger view on this that i'd happily cede to.)

**Comment by jordn** [comment]
agree harry and +1 on defocusing from 'comparison' and more on the 'human review' and 'detail' aspects

**Comment by olehvell-h** [comment]
@harry-humanloop @jordn in the description my take two.  I'm still finding the right tone, let me know what you think

File: fern/changelog/2024-10-02.md (hunk 2 above)
Comment (jordn) [reply fern/changelog/2024-10-02.md:5]: why is the scrollbar on the LHS so much bigger? It's quite ugly.

File: fern/changelog/2024-10-02.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## Evals Comparison Mode
+
+We've added Comparison Mode to the Evals page. Comparison Mode enables Domain Experts to view multiple outputs side-by-side and annotate generations with easy-to-use controls.
```
Comment (PeadarOhAodha) [inline fern/changelog/2024-10-02.md:3]: would avoid introducing 'annotate' here

**Review by PeadarOhAodha — APPROVED** [review]

**Pull request merged**